src/features/classification/security-controls/SecurityControlsModal.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../../styles/variables';
+
+.bdl-SecurityControlsModal {
+    .link {
+        color: $bdl-box-blue !important;
+        text-decoration: none;
+
+        &:hover {
```

Comment:
Does this need a `&:focus` rule, as well?

Reply:
removed